modules/panorama/main.tf

```diff
@@ -55,7 +55,7 @@ resource "azurerm_virtual_machine" "panorama" {
     name              = coalesce(var.os_disk_name, "${var.panorama_name}-disk")
     caching           = "ReadWrite"
     create_option     = "FromImage"
-    managed_disk_type = "Standard_LRS"
+    managed_disk_type = var.panorama_disk_type
```

Comment:
shouldn't we add the same functionality to managed disks? For the moment the disk type is hardcoded [there](https://github.com/PaloAltoNetworks/terraform-azurerm-vmseries-modules/blob/develop/modules/panorama/main.tf#L96).

Reply:
I add Some logic, but there is a catch - since we add disks via map I add logic to specify disk type for each disks, but I drop validation here (It can look too complex in one line).